TUM-Dev/NavigaTUM
# Adjusted Feedback Disclaimer

Fixes #436 

## Proposed Changes (include Screenshots if possible)

- Made the font be the same size as the other disclaimer
- fixed the typo

## How to test this PR

1. See this in the staging (hopefully, crosses fingers)
2. (otherwise launch it locally)

## How has this been tested?

/

## Checklist:
